openedx/core/djangoapps/credit/tests/test_views.py

```diff
@@ -334,3 +335,106 @@ def _assert_request_status(self, uuid, expected_status):
         """
         request = CreditRequest.objects.get(uuid=uuid)
         self.assertEqual(request.status, expected_status)
+
+
+class CreditCourseViewSetTests(TestCase):
+    """ Tests for the CreditCourse endpoints.
+
+     GET/POST /api/v1/credit/creditcourse/
+     GET/PUT  /api/v1/credit/creditcourse/:course_id/
+    """
+    path = reverse('credit:creditcourse-list')
+
+    def setUp(self):
+        super(CreditCourseViewSetTests, self).setUp()
+
+        # Create a user and login, so that we can use session auth for the
+        # tests that aren't specifically testing authentication or authorization.
+        password = 'password'
```

Comment:
Minor: there's no need to declare this variable before every test case. Make it a class variable, or just don't use a variable?